bigquery/src/main/java/org/apache/iceberg/gcp/bigquery/BigQueryMetastoreUtils.java

```diff
@@ -0,0 +1,97 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.iceberg.gcp.bigquery;
+
+import com.google.api.services.bigquery.model.ExternalCatalogDatasetOptions;
+import com.google.api.services.bigquery.model.ExternalCatalogTableOptions;
+import com.google.api.services.bigquery.model.SerDeInfo;
+import com.google.api.services.bigquery.model.StorageDescriptor;
+import java.util.Map;
+
+/** Shared utilities for BigQuery Metastore specific functions and constants. */
+public final class BigQueryMetastoreUtils {
+
+  private BigQueryMetastoreUtils() {}
+
+  public static final String HIVE_SERIALIZATION_LIBRARY =
+      "org.apache.iceberg.mr.hive.HiveIcebergSerDe";
+  public static final String HIVE_FILE_INPUT_FORMAT =
+      "org.apache.iceberg.mr.hive.HiveIcebergInputFormat";
+  public static final String HIVE_FILE_OUTPUT_FORMAT =
+      "org.apache.iceberg.mr.hive.HiveIcebergOutputFormat";
+
+  // TODO(b/341933455): Consider using "org.apache.iceberg.mr.hive.HiveIcebergSerDe" when
```

Comment:
TBH I find those TODOs with that custom tag a bit confusing for readers of the code as it's like adding references to internal JIRA tickets. I understand that this is meant for Google engineers but I don't think stuff like this should be added to Iceberg becuse only a subset of people know what it actually refers to.

Reply:
I removed all places.